RELEASE-MASTERS.md

```diff
@@ -1,10 +1,85 @@
-# Releasing Knative
+# Release Masters
+
+For each release cycle, we dedicate a team of two individuals, one from Eventing
+and one from Serving, to shepherd the release process. Participation is
+voluntary and based on good faith. We are only expected to participate during
+our local office hour.
+
+# Roster
+
+We seed this rotation with all approvers from all the Serving and Eventing
+workgroups, excluding productivity. If you are no longer active in Knative, or
```

Comment:
Why do we exclude productivity?
For the oncall rotation it made sense, but here I don't really see the reason why — and it'd give us more people.

Reply:
@chaodaiG @chizhg Do you have an issue with this? If not I can divide the productivity approvers between the Serving and the Eventing side.

I think this suggestion is good since the involvement of productivity work group will give ideas for automation.

Reply:
We (productivity WG) have oncall rotation - https://knative.github.io/test-infra/, so whoever is oncall during the release dates should be taking care of the release automation issues.

Reply:
I see. Let's keep the productivity rotation separated in that case.